bindings/aws/dynamodb/dynamodb.go

```diff
@@ -95,13 +94,8 @@ func (d *DynamoDB) Invoke(ctx context.Context, req *bindings.InvokeRequest) (*bi
 }
 
 func (d *DynamoDB) getDynamoDBMetadata(spec bindings.Metadata) (*dynamoDBMetadata, error) {
-	b, err := json.Marshal(spec.Properties)
-	if err != nil {
-		return nil, err
-	}
-
 	var meta dynamoDBMetadata
-	err = json.Unmarshal(b, &meta)
+	err := mapstructure.WeakDecode(spec.Properties, &meta)
```

Comment:
Same as the comment on the other PR - why is this necessary?

Reply:
Not strictly necessary, but avoids marshalling to JSON and back. It's just more efficient (and the module was already imported in Dapr).